app/views/projects/show.html.erb

```diff
@@ -0,0 +1,6 @@
+<h1><%= @project.name %></h1>
+<h3>Material: <%= @project.material %></h3>
+<h3>Challenge Theme: <%= @project.challenge.theme %></h3>
+
+<h4>Number of Contestants: <%= @project.contestants.count %></h4>
+<h4>Average Contestant Experience: <%= @average_experience %></h4>
```

Comment:
It's also okay to call the `average_experience` on the @project instance here in the view:
`<h4>Average Contestant Experience: <%= @project.average_experience %></h4>`

You don't necessarily need another instance variable here, but it's not wrong to have it. It's totally okay to call model instance methods that you create in the view, we just don't want to call the AR methods directly.